compechem/core/base.py

```diff
@@ -0,0 +1,14 @@
+class BaseEngine:
```

Comment:
Toglierei "Base" dal nome della classe, Engine è già sufficiente :)

Reply:
Changed name from `BaseEngine` to `Engine`, updated code, docs, and tests